lib/util.js

```diff
@@ -1,5 +1,6 @@
-import fs from 'node:fs';
+import fs, { close, openSync, statSync, utimesSync } from 'node:fs';
```

Comment:
Maybe good for consistency to either always use `fs` or the named exports?

Reply:
Yup